### DISCUSSION TITLE
How set a border radius with decorations to false

### DISCUSSION THREAD

**Discussion by killix** [tauri-apps/tauri#3967] [Q&A]
Hi,

I would like to have a window without decorations (`tauri.config.window.decorations = false`) but with the `border-radius`.
Basically, my goal is just to hide the `titlebar` and keep the window border.
How I can do it?

Thanks for your help.

**Comment by zhanglun** [comment]
similar issues, see [this discussion](https://github.com/tauri-apps/tauri/discussions/3219)

**Comment by wkl007** [comment]
https://github.com/tauri-apps/window-shadows
